src/main/kotlin/no/nav/familie/ef/personhendelse/handler/PersonhendelseHandler.kt

```diff
@@ -55,11 +74,58 @@ abstract class PersonhendelseHandler(
         val oppgaveBeskrivelse = lagOppgaveBeskrivelse(personhendelse)
         val opprettOppgaveRequest = defaultOpprettOppgaveRequest(personIdent, "Personhendelse: $oppgaveBeskrivelse")
         val oppgaveId = oppgaveClient.opprettOppgave(opprettOppgaveRequest)
+        personhendelseRepository.lagrePersonhendelse(UUID.fromString(personhendelse.hendelseId), oppgaveId, personhendelse.endringstype)
         logger.info("Oppgave opprettet med oppgaveId=$oppgaveId")
     }
 
-    open fun skalOppretteOppgave(personhendelse: Personhendelse) = true
+    private fun hentOppgave(personhendelse: Personhendelse): Oppgave {
+        val hendelse = personhendelseRepository.hentHendelse(UUID.fromString(personhendelse.tidligereHendelseId))
+        return oppgaveClient.finnOppgaveMedId(hendelse!!.oppgaveId)
+    }
 
-    abstract fun lagOppgaveBeskrivelse(personhendelse: Personhendelse): String
+    private fun annullerEllerKorrigerOppgave(personhendelse: Personhendelse) {
+        val oppgave = hentOppgave(personhendelse)
```

Comment:
Si eks att det kommer en hendelse på en person før vi har vedtatt noe på den, sedan fatter vi ett vedtak på personen og så kommer det en korrigering/annulering.

Sånn att hvis vi ikke har treff i databasen, så burde vi kanskje sjekke om personen finnes i ef-sak likevel, og då opprette en oppgave for personen?